TLPhotoPicker/Classes/TLAssetsCollection.swift

```diff
@@ -201,10 +203,6 @@ public struct TLPHAsset {
             }
             return PHImageManager.default().requestImageData(for: phAsset, options: requestOptions, resultHandler: { (data, uti, orientation, info) in
                 do {
-                    var data = data
-                    if convertLivePhotosToJPG == true, let imgData = data, let rawImage = UIImage(data: imgData)?.upOrientationImage() {
-                        data = rawImage.jpegData(compressionQuality: 1)
```

Comment:
Even if this is not live photo, and convertLivePhotosToJPG is true, this tries to create jpeg data which is inefficient.

Reply:
I'd like to suggest this.
```swift3
let needConvertLivePhotoToJPG = phAsset.mediaSubtypes.contains(.photoLive) == true && convertLivePhotosToJPG == true
                    if needConvertLivePhotoToJPG, let imgData = data, let rawImage = UIImage(data: imgData)?.upOrientationImage() {
                        data = rawImage.jpegData(compressionQuality: 1)
                    }
```

Reply:
jpegData(compressionQuality: 1) also strips off meta data of photo ( [more here)](https://stackoverflow.com/questions/17475392/when-should-i-use-uiimagejpegrepresentation-and-uiimagepngrepresentation-for-upl) . I'd suggest that data returned with requestImageData should be written to disk and used. Fixing orientation should be not be done by TLPhotoPicker. It should be done by apps using it.

Reply:
Actually you can other way what you want.
Extension implementation is not for TLPhotoPicker.
just convenience methods.


https://github.com/tilltue/TLPhotoPicker/blob/master/README.md
>  Note:  convenience export method
>  fullResolutionImage, cloudImageDownload, tempCopyMediaFile, exportVideoFile
>  It's not enough if you wanted to use more complicated export asset options. ( progress, export type, etc..)

Reply:
All I'm saying is, If these changes are applied, previous users will get different results.

Reply:
Yes, it make sense. This can be done with extension, and this change may break for old users. How about fixing the issue with wrong  convertLivePhotosToJPG.
with this code- `let needConvertLivePhotoToJPG = phAsset.mediaSubtypes.contains(.photoLive) == true && convertLivePhotosToJPG == true
                    if needConvertLivePhotoToJPG, let imgData = data, let rawImage = UIImage(data: imgData)?.upOrientationImage() {
                        data = rawImage.jpegData(compressionQuality: 1)
                    }`